core/scripts/vrfv2/testnet/main.go

```diff
@@ -21,6 +21,8 @@ import (
 	"github.com/shopspring/decimal"
 
 	"github.com/smartcontractkit/chainlink/core/assets"
+	"github.com/smartcontractkit/chainlink/core/internal/gethwrappers/generated/vrfv2_wrapper"
+	"github.com/smartcontractkit/chainlink/core/internal/gethwrappers/generated/vrfv2_wrapper_consumer_example"
```

Comment:
Small nit: I think these imports are in the wrong place (should be grouped with the other gethwrappers stuff further down).

Reply:
fixed!